src/configure/clients/github/TemplateServiceClient.ts

```diff
@@ -1,19 +1,505 @@
+                    "properties": {
+
+                    },
+                    "inputMode": 10,
+                    "dataSourceId": null,
+                    "defaultValue": null,
+                    "staticValidation": null,
+                    "dynamicValidations": [],
+                    "visibleRule": null,
+                    "id": "dockerFilePath",
+                    "description": "The path to the Dockerfile used for building the container image. Minimatch patterns are supported, for example: **/Dockerfile. The complete path from the repository root to the Dockerfile can also be used. For example: Fabrikam/WebApp/Dockerfile.",
+                    "type": 0,
+                    "possibleValues": []
+                },
+                {
+                    "name": "PORT",
+                    "groupId": "dockerInputs",
+                    "isRequired": false,
+                    "sublabel": null,
+                    "properties": {
+                        
+                    },
+                    "inputMode": 10,
+                    "dataSourceId": null,
+                    "defaultValue": "",
+                    "staticValidation": null,
+                    "dynamicValidations": [],
+                    "visibleRule": null,
+                    "id": "containerPort",
+                    "description": "Specify port to expose in container.",
+                    "type": 0,
+                    "possibleValues": []
+                },
+                {
+                    "name": "Docker build context",
+                    "groupId": "dockerInputs",
+                    "isRequired": false,
+                    "sublabel": null,
+                    "properties": {},
+                    "inputMode": 10,
+                    "dataSourceId": null,
+                    "defaultValue": "",
+                    "staticValidation": null,
+                    "dynamicValidations": [],
+                    "visibleRule": null,
+                    "id": "dockerContext",
+                    "description": "Specify the source directory path which will be used for building container.",
+                    "type": "string",
+                    "possibleValues": []
+                }
+            ],
+            "id": "ms.vss-continuous-delivery-pipeline-templates.aks-cd-with-dockerfile-github",
+            "description": "i18n:Template for configuring github workflow on an AKS cluster using dockerfile",
+            "attributes": {
+                "language": "Docker",
+                "buildTarget": "Dockerfile",
+                "deployTarget": "Azure:AKS",
+                "serviceId": "ms.vss-continuous-delivery-pipeline-templates.service-type-aks",
+                "templateType": "continuous-delivery",
+                "pipelineType": "github",
+                "codeRepositoryType": "BYOC"
+            }
+        };
+
+        return extendedPipelineTemplate2 || extendedPipelineTemplate;
```

Comment:
nit: just return extendedPipelineTemplate2

Reply:
removed the hardcoded template and saved it in my local for future reference. now returning extendedPipelineTemplate